### PR TITLE
Display safe buttons

### PR DESCRIPTION
**Pull request by Mati0x**
#366

### CODE REVIEW

File: apps/web/components/PoolHeader.tsx [hunk 1]
```diff
@@ -191,20 +201,39 @@ export default function PoolHeader({
...
+
+  const disableCouncilSafeButtons = disableCouncilSafeBtnCondition.some(
+    (cond) => cond.condition,
+  );
+
+  const { tooltipMessage, missmatchUrl, isConnected } = useDisableButtons(
+    disableCouncilSafeBtnCondition,
+  );
+
+  //
```
Comment (Corantin) [inline apps/web/components/PoolHeader.tsx:220]:
```suggestion
```

File: apps/web/components/PoolHeader.tsx [hunk 2]
```diff
@@ -156,6 +156,16 @@ export default function PoolHeader({
...
     : poolConfig;
 
+  //hooks
+  const { data: isCouncilMember } = useContractRead({
+    address: strategy.registryCommunity.councilSafe as Address,
+    abi: abiWithErrors2(safeABI),
+    functionName: "isOwner",
+    chainId: Number(chainId),
+    enabled: !!address,
+    args: [address?.toLowerCase() as Address],
```
Comment (Corantin) [inline apps/web/components/PoolHeader.tsx:166]: you really need lowercase here ?

**Review by Corantin — DISMISSED** [review]

File: apps/web/components/PoolHeader.tsx (hunk 2 above)
Comment (Mati0x) [reply apps/web/components/PoolHeader.tsx:166]: I guess not, nice catch!

File: apps/web/components/PoolHeader.tsx [hunk 3]
```diff
@@ -157,6 +157,16 @@ export default function PoolHeader({
...
     : poolConfig;
 
+  //hooks
+  const { data: isCouncilMember } = useContractRead({
+    address: strategy.registryCommunity.councilSafe as Address,
+    abi: abiWithErrors2(safeABI),
+    functionName: "isOwner",
+    chainId: Number(chainId),
+    enabled: !!address,
+    args: [address as Address],
```
Comment (Corantin) [inline apps/web/components/PoolHeader.tsx:167]:
```suggestion
    args: [address as Address],
    onError: () => {
      console.error("Error reading isOwner from Coucil Safe");
    },
```

File: apps/web/components/DisputeButton.tsx [hunk 4]
```diff
@@ -134,7 +137,16 @@ export const DisputeButton: FC<Props> = ({ proposalData }) => {
...
-  const isCouncilSafe = config.tribunalSafe === address?.toLowerCase();
+  const isTribunalSafe = config.tribunalSafe === address?.toLowerCase();
+
+  const { data: isTribunalMember } = useContractRead({
+    address: config.tribunalSafe as Address,
+    abi: abiWithErrors2(safeABI),
+    functionName: "isOwner",
+    chainId: Number(chainId),
+    enabled: !!address,
+    args: [address as Address],
```
Comment (Corantin) [inline apps/web/components/DisputeButton.tsx:148]:
```suggestion
    args: [address as Address],
    onError: () => {
      console.error("Error reading isOwner from Tribunal Safe");
    },
```

**Review by Corantin — APPROVED** [review]

**Review by Corantin — APPROVED** [review]

**Pull request merged**